.github/coderabbit/release-pr-prompt.md

```
@@ -0,0 +1,34 @@
+## 🕵️ Release-Gate Audit: Breaking Changes & Architecture
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)